torch/fx/experimental/symbolic_shapes.py

```diff
@@ -756,6 +761,31 @@ def __post_init__(self):
         if self.constraint_sizes is None:
             object.__setattr__(self, 'constraint_sizes', [None] * len(self.dynamic_sizes))
 
+
+@dataclass(frozen=True)
+class KnownCreateSymbolicPolicy(FreshCreateSymbolicPolicy):
```

Comment:
we should probably rename these classes since they're not really policy anymore

Reply:
How about StatelessSymbolicContext and StatefulSymbolicContext

Reply:
:)